api/app/signals/apps/api/views/signals/public/signals.py

```diff
@@ -107,7 +107,7 @@ def geography(self, request) -> Response:
         ).exclude(
             # Only signals that are in an "Open" state
             status__state__in=[AFGEHANDELD, AFGEHANDELD_EXTERN, GEANNULEERD, VERZOEK_TOT_HEROPENEN],
-
+        ).exclude(
```

Comment:
In plaatsvan een dubbele exclude te gebruiken maar en een OR statement van:
```
.exclude(
            # Only signals that are in an "Open" state
            # Only Signal's that are in categories that are publicly accessible
            Q(status__state__in=[AFGEHANDELD, AFGEHANDELD_EXTERN, GEANNULEERD, VERZOEK_TOT_HEROPENEN]) |
            Q(category_assignment__category__is_public_accessible=False)
        )
```

Reply:
Thanks for the quick review, will implement as suggested.